Readme.md

```diff
@@ -110,6 +110,19 @@ Rspec: Add to your `.rspec_parallel` (or `.rspec`) :
 
     --format progress
     --format ParallelTests::RSpec::RuntimeLogger --out tmp/parallel_runtime_rspec.log
+    
+If your runtime log file path is different from `tmp/parallel_runtime_spec.log`, you **MUST** use `parallel_test` CLI (instead of rake task) and set `--runtime-log` option:
+
+```
+# .rspec_parallel
+--format progress
+--format ParallelTests::RSpec::RuntimeLogger --out coverage/parallel_runtime_rspec.log
+```
+
+```
+# run test suite
+parallel_test spec -t rspec --runtime-log coverage/parallel_runtime_rspec.log
+```
```

Comment:
can you shorten this to `To use a custom logfile location use the CLI: "parallel_test spec -t rspec --runtime-log my.log"`